charts/tekton-apps/templates/roles.yaml

```diff
@@ -14,7 +21,7 @@ metadata:
  namespace: {{ $.Release.Namespace }}
 rules:
   - apiGroups:  ["extensions", "apps", "batch", ""]
-    resources:  ["services", "deployments", "pods", "jobs"]
+    resources:  ["services", "deployments", "pods", "jobs", "pods/log"]
```

Comment:
doesn't look right to me. Why not to use the previous role?

Reply:
done, added a separate `roles.yaml` file in `wordpress` folder